Workshop.md

```diff
@@ -466,6 +466,9 @@ All sixteen tests should now be passing!
 
 Now try visiting [https://todobackend.com/client/](https://todobackend.com/client/) in your browser to view the ToDo-Backend web client. Enter an API root of `http://localhost:8080/` and use the website to interact with your REST API. You can add, remove and update ToDo items as you wish.
 
+### Attach a database to your project
+This [extra tutorial](https://github.com/IBM/ToDoBackend/blob/master/DatabaseWorkshop.md) builds upon the project created in this Workshop and replaces storing ToDos in an Array with a database running locally on your machine, using Swift-Kuery-ORM and PostgreSQL. 
```

Comment:
I don't like the name "extra tutorial" how about "ORM tutorial" or "database tutorial"?

Reply:
✅